librad/src/paths.rs

```diff
@@ -7,38 +8,92 @@ use std::{
 use directories::ProjectDirs;
 
 #[derive(Clone)]
-pub struct Paths(ProjectDirs);
+pub struct Paths {
+    pub keys_dir: PathBuf,
```

Comment:
Is it ok for users to manipulate a constructed `Paths`?

Reply:
Fair